packages/tasit-action/src/TasitAction.js

```diff
@@ -216,6 +223,11 @@ class TransactionSubscription extends Subscription {
     const tx = await this.#txPromise;
     await this.#provider.waitForTransaction(tx.hash);
   };
+
+  // For testing purposes
```

Comment:
Do we already have a thread somewhere tracking what we want to do with functions that are only used for testing? I think we do, in which case feel free to resolve this one.

Reply:
Yes, here: https://github.com/tasitlabs/TasitSDK/pull/102#discussion_r247713587